### ISSUE TITLE
[feature] - JDBC getImportedKeys method support

### ISSUE THREAD

**Issue by archenroot** [omnisci/mapd-core#310]
I am trying to integrate with standard DB IDEs and facing small issue:
```
Not supported yet, line:1388 class:com.mapd.jdbc.MapDDatabaseMetaData method:getImportedKeys
Not supported yet, line:1388 class:com.mapd.jdbc.MapDDatabaseMetaData method:getImportedKeys
```
Do you plan to support such JDBC interface method in near future?

Thank you for response.

**Comment by cdessanti** [comment]
Do you mean DBeaver? It's trivial to add support to such methods

Ottieni Outlook per Android<https://aka.ms/ghei36>

________________________________

**Comment by archenroot** [comment]
Yeah, sure, I will make just empty ResultSet as mapd doesn't work with primary neither foreign keys. But these methods are used by other tools as well.... so just to make them not scream.

**Comment by randyzwitch** [comment]
Features like these are great areas for community contributions, so if you have the capability to add this functionality, we'd be happy to incorporate it.

**Comment by cdessanti** [comment]
Ok, I'll add the generation of an empty result set with the correct names and datatype on those methods.
